modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/generalized_plane_strain.md

```diff
@@ -139,7 +141,217 @@ N_{z}
 
 The off-diagonal entries are nonzero, but not shown here.
 
-### List of MOOSE objects used in the implementation
+## $x$-$z$ plane generalized plane strain problem
+
+### In-plane equilibrium equations
```

Comment:
This seems like way too much detail to me for the xz and yz planes, and much of this is essentially duplicated from above. This is how I'd do it:
```
## $x$-$z$ and $y$-$z$ plane generalized plane strain problem
The generalized plane strain formulation can also be used if the two-dimensional model is represented in the $x$-$z$ or $y$-$z$ planes, rather than the $x$-$y$ plane, as is typically the case. If the model lies in those other planes, the calculation of the strain tensor is modified to take into account the fact that the model is in a different plane. Also, the scalar variable used to represent the out-of-plane strain in the generalized plane strain formulation is in a different direction. All other aspects of the formulation are identical to the $x$-$y$ plane case.

For the case when the model lies in the $x$-$z$ plane, the small-strain kinematic equations for the strain calculation that are equivalent to [insert reference to xy plane case] for the $x$-$y$ plane are expressed as:
\begin{equation}
\left\{\begin{matrix}
\epsilon_{xx} \\
\epsilon_{zz} \\
\gamma_{xz}
\end{matrix}\right\}
=
\left\{\begin{matrix}
\frac{\partial u}{\partial x} \\
\frac{\partial w}{\partial z} \\
\frac{\partial w}{\partial x} + \frac{\partial u}{\partial z}
\end{matrix}\right\}
\end{equation}
The generalized plane strain equilibrium equation equivalent to [insert reference to xy equivalent] is expressed as:
\begin{equation}
N_{y} = \int_{A}{\sigma_{yy}dA} = \bar{N}_{y}
\end{equation}
The same pattern is followed for the $y$-$z$ plane case.